.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
+*.iml
```

Comment:
Why do you need `iml` at the same time you have `.idea` folder here?

Reply:
removed